pkg/daemon/ceph/client/radosnamespace.go

```diff
@@ -104,3 +104,32 @@ func DeleteRadosNamespace(context *clusterd.Context, clusterInfo *ClusterInfo, p
 	logger.Infof("successfully deleted rados namespace %s/%s in k8s namespace %q", poolName, namespaceName, clusterInfo.Namespace)
 	return nil
 }
+
+// EnableRBDRadosNamespaceMirroring enables rbd mirroring on a rados namespace.
+func EnableRBDRadosNamespaceMirroring(context *clusterd.Context, clusterInfo *ClusterInfo, poolAndRadosNamespaceName, remoteNamespace, mode string) error {
+	logger.Infof("enable mirroring in rados namespace %s in k8s namespace %q", poolAndRadosNamespaceName, clusterInfo.Namespace)
+	args := []string{"mirror", "pool", "enable", poolAndRadosNamespaceName, mode, "--remote-namespace", remoteNamespace}
```

Comment:
This command will fail if it's not a new ceph image with the new command, right? Need to add a version check still, or did I miss it?

Reply:
Yes correct, for this whole feature we need version check